live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -233,3 +233,42 @@
   with_items:
     - { key: "net.ipv4.conf.all.promote_secondaries", value: "1" }
     - { key: "net.ipv4.conf.default.promote_secondaries", value: "1" }
+
+#
+# Configure command audit logging
+#
+# We want to record all commands executed on the appliance. Opt out for
+# setsid since all ExecuteUtils.execute wrap each call with setsid.
+#
+# Note we can't use the "service" module, as that fails when running
+# inside the chroot environment, so we enable auditd using "command".
+#
+- apt:
+    name: auditd
+    state: installed
+    update_cache: no
+
+- command: /bin/systemctl enable auditd.service
```

Comment:
Can we do this with the [service module](https://docs.ansible.com/ansible/latest/modules/service_module.html)? or does that not work when using a chroot like we're doing?

Reply:
Doesn't seem to work as a service module in our context.